template.html

```diff
@@ -7,7 +7,7 @@
     <meta property="og:url" content="http://www.etcher.io" />
     <meta property="og:type" content="website" />
     <meta property="og:title" content="Etcher by Resin.io" />
-    <meta property="og:description" content="Burn images to SD cards and USB drives, safe and easy." />
+    <meta property="og:description" content="Burn images to SD cards & USB drives, safely and easily." />
```

Comment:
In the spirit of DRY, could this also come from `data.json`? Or doesn't that make sense in this case?